packages/api/test/e2e/api/promise-tx-eras.spec.ts

```diff
@@ -74,7 +74,7 @@ describeE2E({
     });
 
     it('fails on a transfer with invalid time', async (done): Promise<void> => {
-      const nonce = await api.query.system.accountNonce(keyring.alice.address) as Index;
+      const nonce = await api.query.system.accountNonce(keyring.alice.address);
```

Comment:
I did a couple of these in the codebase, end-users would need to do this too, or else eslint complains "assertion is useless"